src/hooks/http.js

```diff
@@ -219,25 +232,34 @@ export const httpRequestWrapper = originalRequestFn =>
         const traceId = getPatchedTraceId(awsXAmznTraceId);
         options.headers['X-Amzn-Trace-Id'] = traceId;
       }
-
       const requestData = parseHttpRequestOptions(options, url);
+      const requestRandomId = getRandomId();
```

Comment:
I don't understand the significance of `requestRandomId` in relation to the timer